generators/generator-bot-core-assistant/generators/app/templates/README.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to your new bot
 
-This Bot Project was created using the Assistant Core template. Your bot is an assistant-style bot - it is designed as a base for a bot that will help your users accomplish multiple different tasks.
+This Bot Project was created using the Core Assistant template. Your bot is an assistant-style bot - it is designed as a base for a bot that will help your users accomplish multiple different tasks.
```

Comment:
I don't believe "bot project" needs to be capitalized.

Reply:
you're right.  changed this casing here and in every other generated README